### PR TITLE
messaging list

### PR DESCRIPTION
**Pull request by PablitoGS**
component messaging list for chat cells, please review @nguasch

### CODE REVIEW

File: html/messaging.html [hunk 1]
```diff
@@ -8,33 +8,81 @@
...
-    <div>
-      <p class="messaging-title truncate">Desarrollador/Analista de aplicaciones web</p>
-      <p class="messaging-status-chat truncate">Julia: Hooola</p>
+    <div class="messaging-avatar">
+      <img class="crop-circle-xsmall border-default" alt="Nombre Apellidos" src="https://media.infojobs.net/app/pictures/pic-view-cv-default-photo.png">
+    </div>
+    <div class="messaging-body">
+      <div class="messaging-body-main">
+        <p class="messaging-title truncate">UX / UI Designer</p>
+        <p class="messaging-meta">04/09/16</p>
```
Comment (nguasch) [inline html/messaging.html]: You could use time tag with attribute datetime

File: html/messaging.html [hunk 2]
```diff
@@ -8,33 +8,81 @@
...
+      <div class="messaging-body-secondary">
+        <p class="messaging-last-text truncate">Julia: a buscar curro en InfoJobs todos los días, a buscar curro en InfoJobs todos los días, a buscar curro en InfoJobs todos los días, a buscar curro en InfoJobs todos los días</p>
+      </div>
+    </div>
+    <div class="messaging-icons-right">
+      <p class="iconfont-Arrow focus" title="Ir al chat"><span class="hide">Ir al chat</span></p>
+    </div>
+  </li>
+  <li class="messaging-list-item">
+    <div class="messaging-icons-left">
```
Comment (nguasch) [inline html/messaging.html]: You must add hide-large-device class and medium-large-device to hide this icon

**Comment by nguasch** [comment]
You must add margins for date and messaging-icons-left when messaging-title is too long

**Comment by PablitoGS** [comment]
@nguasch fixed

File: styles/components/messaging.scss [hunk 3]
```diff
@@ -56,6 +130,11 @@ Styleguide #{$sgi-messaging}
       color: $soft-script;
     }
 
+    .messaging-meta,
+    .sticker {
+      margin-left: 10px;
```
Comment (nguasch) [inline styles/components/messaging.scss]: it's better to use margin-left: $padding/2; ?

**Comment by PablitoGS** [comment]
I've added active status and fixed paddings @nguasch

**Comment by nguasch** [comment]
👍

**Pull request merged**